### DISCUSSION TITLE
ISCE Ionosphere Correction TerraSAR-X collected in Florida for stripmapstack to Stamps

### DISCUSSION THREAD

**Discussion by tonianr** [isce-framework/isce2#438] [Q&A] [answered]
Hello -- I am processing some TSX scenes collected in Florida (small, about 6x12 km). I've been wondering if it would be ideal to complete an ionosphere correction in the ISCE stackstripmap step, given how small my area is? 

Below is an image of PS deformation from an installed corner reflector in my study area:
![CRs_north_IDs155387_2339](https://user-images.githubusercontent.com/78233913/152233147-e73c8e7a-5f83-4de5-aa24-b5b063703b92.png)

The blue is my work using ISCE to StaMPS processing, while the orange was processed by a colleague using DORIS to StaMPS. I am attempting to reprocess the data using ISCE and with some newer scenes.

Before adjusting parameters in StaMPS, I want to make sure I'm preparing the data well enough for PS processing.
My focus is to monitor mmm scale movement on buildings in FL.

Is there a chance that the ionosphere correction in ISCE will help reduce noise in my persistent scatterers?

Also, I'd appreciate any pointers on using the parameters available for the ionosphere correction ins ISCE.

**Comment by hfattahi** [comment, marked as answer]
Hi @tonianr , Before digging into more details or reprocessing your data, would you please confirm that your two time-series plots above are referenced to the exact same reference pixel? 
Basically such comparison is only meaningful if the reference pixel is exactly the same between the two datasets.

Also you may check and make sure that your stamps processing chains are identical between the two datasets. Any atmosphere correction or ramp removal can change the time-series and make one less or more noisy than the other. I'm not familiar with Stamps. But if it allows you to turn off all corrections for both datasets, that would be the first check to compare the two time-series. 

I don't think if you need ionosphere correction for your application. Ionosphere impact on SAR phase is inversely proportional to the square of frequency. In X-band it is ~50 times less than L-band. Also it looks like you are interested in very localized signals on buildings. Ionosphere in regions like Florida introduces very spatially long-wavelength phase patterns which does not matter when you look at the time-series between the two pixels few km apart. 

Besides phase, ionosphere also introduces geolocation errors which I don't expect to be a show stopping for your X-band dataset. 

I think for your application you will have other dominant sources of errors including DEM errors as the height of the buildings are not in the existing DEMs.

**Reply by tonianr** [reply]
Hi @hfattahi 
Thank you for responding.

I can confirm that the two plots are from one of the same corner reflectors installed in our survey area since it is grassy and they are the only reliable scatterers in the area.

My dilemma is that the processing files for the orange dataset are not available anymore and my only means of comparing data is the time series output from StaMPS. 

Both datasets were processed with SRTM DEMs, and I intend to reprocess with DSMs created by LiDAR data for my research. But for now, I am tasked to replicate a less noisy time series plot (from the Corner reflectors), similar to the orange plot above!

I also have a ~11 m shift in my dataset, but it is only noticeable when zoomed into the survey area.

I'll take your advice and leave my isce processing as is with no filters/corrections and focus on the StaMPS processing chain and atmospheric corrections.

**Comment by hfattahi** [comment]
I understand that the time-series plots are for the same corner reflector. But are they referenced to the same reference pixel? Note that InSAR gives you relative displacement. You always need a reference pixel. Some components of noise at your Corner Reflector is a function of the distance between the corner reflector and your reference pixel. If your orange plot has a reference pixel close by and your blue plot has a reference pixel far away, then sure blue is going to be noisier. For a fair comparison they must be relative to the same reference pixels. Hope that is clear.

For your application the height of the building is important. If you have a DSM, then yes reprocessing with a DSM is going to be helpful. It reduces DEM error effects on your time-series and it reduces geolocation error (your 11 m shift is most likely due to the height error of the building.)

**Reply by tonianr** [reply]
Thank you, I understand now. I realize that I've entirely overlooked setting a reference pixel in StaMPS... I will ask for information on the reference pixel location for the orange dataset!
